WcaOnRails/lib/competition_results_validator.rb

```diff
@@ -147,83 +128,6 @@ def merge(other_validators)
     @total_warnings = @warnings.values.sum(&:size)
   end
 
-  def results_similar_to(reference, reference_index, results)
```

Comment:
Note: this is unrelated, but this is dead code I forgot to remove when doing #4357.